docs/astro.config.mjs

```diff
@@ -34,72 +34,47 @@ export default defineConfig({
           collapsed: false,
           items: [
             { label: 'Overview', link: '/' },
-            { label: 'Why XTDB?', link: '/intro/why-xtdb' },
             { label: 'Getting started', link: '/intro/getting-started' },
-            {
-              label: 'What is XTDB?',
-              items: [
-                { label: 'At a glance', link: '/intro/what-is-xtdb' },
-                { label: 'What is XTQL?', link: '/intro/what-is-xtql' },
-                { label: 'Data model', link: '/intro/data-model' },
-                // { label: 'Architecture', link: '/intro/architecture' },
-                // { label: 'Bitemporality', link: '/intro/bitemporality' }
-              ]
-            },
+            { label: 'What is XTDB', link: '/intro/what-is-xtdb' },
+            { label: 'Data model', link: '/intro/data-model' },
             { label: 'Community', link: '/intro/community' },
             { label: 'Roadmap', link: '/intro/roadmap' },
+            { label: 'Mission', link: '/intro/why-xtdb' },
```

Comment:
why move 'mission' to the bottom here? makes sense to start with the highest level and read down into the details?

Reply:
the thought was to move it until _after_ Getting Started, agreed it's probably too valuable to push all the way down